src/plugins/analysis/device_tree/code/device_tree.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from common_helper_process import execute_shell_command_get_return_code
+
+from analysis.PluginBase import AnalysisBasePlugin
+
+
+class AnalysisPlugin(AnalysisBasePlugin):
+    '''
+    Device Tree Plug-in
+    '''
+    NAME = 'device_tree'
+    DESCRIPTION = 'get the device tree in text from the device tree blob'
+    DEPENDENCIES = []
+    VERSION = '0.1'
+    # MIME_WHITELIST = ['linux/device-tree']
+
+    def __init__(self, plugin_administrator, config=None, recursive=True):
+        self.config = config
+
+        super().__init__(plugin_administrator, config=config,
+                         recursive=recursive, plugin_path=__file__)
+
+    def process_object(self, file_object):
+
+        device_tree = self.execute_device_tree_compiler(file_object.file_path)
+
+    # store the results
```

Comment:
The comment should be on the same indentation level as the rest of the function. It can probably be removed, though, since it is pretty obvious that the results are stored below